content/hi/cloud-native-security.md

```diff
@@ -0,0 +1,18 @@
+---
+title: क्लाउड नेटिव सुरक्षा (Cloud Native Security)
+status: Feedback Appreciated
+category: संकल्पना
+tags:  ["सुरक्षा"]
+---
+
+## यह क्या है
+
+क्लाउड नेटिव सुरक्षा एक दृष्टिकोण है जो क्लाउड नेटिव अनुप्रयोगों में सुरक्षा निर्मित करता है। यह सुनिश्चित करता है कि सुरक्षा विकास से लेकर निर्माण तक संपूर्ण अनुप्रयोग जीवनचक्र का हिस्सा बनी रहे। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा प्रतिरूप के समान मानकों को क्लाउड नेटिव वातावरण के विवरणों, अर्थात् तेज़ी से कोड परिवर्तन और अत्यधिक अल्पकालिक अवसरंचनाओं  को अपनाते हुए सुनिश्चित करने का प्रयास करती है। क्लाउड नेटिव सुरक्षा डेवसेकॉप्स [DevSecOps](/devsecops/) नामक अभ्यास से अत्यधिक संबंधित है।
+
+## समस्या जिस का यह समाधान करता है
+
+पारंपरिक सुरक्षा प्रतिरूप कई मान्यताओं के साथ बनाए गए थे जो अब मान्य नहीं हैं। [क्लाउड नेटिव ऐप](/cloud-native-apps/) (अनुप्रयोग) बार-बार बदलते हैं, बड़ी संख्या में ओपन सोर्स संसाधनों और लाइब्रेरी का उपयोग करते हैं, अक्सर विक्रेता-नियंत्रित बुनियादी ढांचे में चलते हैं, और बुनियादी ढांचे में तेज़ बदलाव के अधीन होते हैं। कोड समीक्षा, लंबे गुणवत्ता आश्वासन चक्र, मेज़बान-आधारित भेद्यता अवलोकन और अंतिम मिनट की सुरक्षा समीक्षा क्लाउड नेटिव अनुप्रयोगों के साथ [स्केल](/scalability/) नहीं की जा सकती हैं। 
```

Comment:
1. मान्य to वैध
2.  why did you use () in अनुप्रयोग
3. @Garima-Negi will it be better to just write the word 'host-based' as it is in hindi or should we localise it?

Reply:
Hi @sayantani11, thanks for the feedback. Here's the progress:
1. Done👍
2.  'अनुप्रयोग' is the Hindi localization for app. I've put in within brackets with the purpose of providing further context to the reader about the technical term "cloud-native apps". I thought to provide additional information to clarify the meaning of the term. What are your thoughts on this?
3. I too agree with you on this, 'मेज़बान-आधारित' would better be replaced by 'host-based' as it is more of a technical term and may be more commonly known in its English form.💯

Reply:
I'd go with host-based instead of मेज़बान-आधारित. 
We can also eliminate अनुप्रयोग IMO.

Reply:
Hi @Garima-Negi! Thanks for the suggestions, I've incorporated them all in the latest commit.